test/loggerConfig_test.cpp

```diff
@@ -147,7 +147,7 @@ void LoggerConfigTest::testLoggerInitObd2Config() {
    CPPUNIT_ASSERT(c->enabled == 0);
 
    for (int i = 0; i < OBD2_CHANNELS; ++i) {
-      CPPUNIT_ASSERT(strlen(c->pids[i].mapping.channel_cfg.label));
+      CPPUNIT_ASSERT(strlen(c->pids[i].mapping.channel_cfg.label)== 0);
```

Comment:
What changed here exactly?

Reply:
There was some old OBDII initialization code that pre-populated the OBDII channel names with values; this was removed separately so the channel name defaulted to empty string. This just acknowledges that change.